providers/aws/resources/aws_dms.go

```diff
@@ -31,7 +30,9 @@ func (a *mqlAwsDms) replicationInstances() ([]interface{}, error) {
 	}
 	// get all the results
 	for i := range poolOfJobs.Jobs {
-		res = append(res, poolOfJobs.Jobs[i].Result.([]types.ReplicationInstance)...)
+		if poolOfJobs.Jobs[i].Result != nil {
```

Comment:
Should we check for an error here and propagate?

Reply:
oh, ya we should! ill do that